airbyte-integrations/connectors/source-hubspot/sample_files/incremental_catalog.json

```diff
@@ -225,17 +225,6 @@
       "cursor_field": ["updatedAt"],
       "destination_sync_mode": "append"
     },
-    {
-      "stream": {
-        "name": "property_history",
-        "json_schema": {},
-        "supported_sync_modes": ["full_refresh", "incremental"],
-        "default_cursor_field": ["timestamp"]
-      },
-      "sync_mode": "incremental",
-      "cursor_field": ["timestamp"],
-      "destination_sync_mode": "overwrite"
-    },
```

Comment:
I noticed you added this to the full_refresh test but then removed it - did we find out why it was causing tests to fail?

Reply:
Yep.. the thing is full_refresh test compares records by the primary key if it is present (not in this case), otherwise it pops ignored fields and makes hashes of them and compares those hashes. Some of the records PropertyHistory stream yields contain current timestamp, so we will never get two identical record sets. Ignoring the fields containing those timestamps makes the test senseless, cause these fields are actually the most valuable part of the record (its type is dynamic, so it does not return *only* timestamp - it varies from record to record)